celements-reference/src/main/java/org/xwiki/model/reference/DocumentReference.java

```diff
@@ -23,17 +23,21 @@
 import java.util.Collections;
 import java.util.List;
 import java.util.Locale;
+import java.util.Optional;
 
 import org.xwiki.model.EntityType;
 
+import one.util.streamex.EntryStream;
+
 /**
  * Represents a reference to a document (wiki, space and page names).
```

Comment:
"(wiki, space and page names)" maybe "language" or Local should be added to this java comment, wdyt?